NSeleneTests/Integration/SharedDriver/SeleneElementJSActionsSpecs.cs

```diff
@@ -0,0 +1,56 @@
+using NUnit.Framework;
+using static NSelene.Selene;
+using NSelene.Tests.Integration.SharedDriver.Harness;
+using NSelene.Conditions;
+using NSelene.Support.Extensions;
+using OpenQA.Selenium;
+
+namespace NSelene.Tests.Integration.SharedDriver
+{
+    [TestFixture]
+    public class SeleneElementJSActionsSpecs : BaseTest
+    {
+        [Test]
+        public void SElementJSScrollIntoView()
+        {
+            // prepare
```

Comment:
here the // prepare clause seems to be redundant, because the following Given word reflect the same meaning...

Also, since we already started to use BDD style with Given/....
Then, instead of // act, it will be more consistent to use // When
and instead of // assert - // Then

But... why to bother?

Let's just start with simple Given telling that we started a "preparation" section, then just use empty lines to divide the whole test into three blocks... it should be obvious that the second block is an "act" and the last one is an "assert", especially taking into account that last block will nevertheless use some methods like assert or should...

Reply:
agree.